Modules/Config/Get-LocalUsers.ps1

```diff
@@ -0,0 +1,17 @@
+<#
+.SYNOPSIS
+Get-LocalUsers.ps1 returns the following details of each local user acccount:
+    - Disabled (bool), 
+    - Name, 
+    - Full Name, 
+    - Caption, 
+    - Domain
+    - SID.  
+
+by the dude with the moe, 18 Nov 2022. 
+.NOTES
+Next line is required by Kansa.ps1. It instructs Kansa how to handle
+the output from this script.
+OUTPUT tsv
```

Comment:
You shouldn't need (nor include) this bit in the collector code anymore. See this blog post re: the `-OutputFormat` command-line parameter, which made this directive in the modules obsolete.